app/controllers/api_controller.rb

```diff
@@ -1,3 +1,8 @@
+#
+# Initializing REST API environment, called once @ startup
+#
+Api::Initializer.new.go
```

Comment:
That's a great question. It will re-"initialize", which is probably not ideal. I'm not sure we can make this a rails initializer though since it touches the database? FWIW it looks like master behaves in the same way.